src/scripts/inclusion-bot.js

```diff
@@ -20,7 +20,10 @@ const getTriggers = () => {
     message,
     triggers: triggers.map(({ ignore, matches, ...rest }) => ({
       ignore: ignore && RegExp(`(${ignore.join("|")})`, "ig"),
-      matches: RegExp(`(${matches.join("|")})`, "i"),
+      matches: RegExp(
+        `(${matches.join("|")})(?=[^"“”']*(["“”'][^"“”']*["“”'][^"“”']*)*$)`,
```

Comment:
Don't ask me how this regex works. I found it a long time ago, back when the Inclusion Bot was just "guys bot," and it worked. It's been hanging out in the code as a comment since then, hoping it'd find its way back into production. And here we are!

Reply:
Something I like to do is link to a regex viewer site in a comment above regexes like this so someone can test it out themselves when they come across it. Like https://regexr.com/ or similar.